tests/io/v0_0_0/test_write.py

```diff
@@ -180,3 +181,54 @@ def test_checksum_on_write(self):
                 loaded = slicedimage.Reader.parse_doc(basename, baseurl)
 
                 loaded.tiles()[0]._load()
+
+    def test_write_tiff(self):
+        image = slicedimage.TileSet(
+            ["x", "y", "ch", "hyb"],
+            {'ch': 2, 'hyb': 2},
+            (100, 100),
+        )
+
+        for hyb in range(2):
+            for ch in range(2):
+                tile = slicedimage.Tile(
+                    {
+                        'x': (0.0, 0.01),
+                        'y': (0.0, 0.01),
+                    },
+                    {
+                        'hyb': hyb,
+                        'ch': ch,
+                    },
+                )
+                tile.numpy_array = np.zeros((100, 100), dtype=np.uint32)
+                tile.numpy_array[hyb, ch] = 1
+                image.add_tile(tile)
+
+        with TemporaryDirectory() as tempdir, \
```

Comment:
A few line comments here would probably be useful as directives for users (make it clear which block is writing, which is reading, and which is comparing).